Changes

```diff
@@ -7,3 +7,9 @@ Revision history for Image::PNG::Portable
 
 0.1  2025-05-23T13:19:10+02:00
     - Initial version as Raku Community Module
+
```

Comment:
Please just add a line of text above the previous timed line: App::Mi6 takes care of putting timestamps in there.

The last upload put:
```
0.2  2025-07-03T22:07:20+02:00
    - Add read & get functions, Matt Doughty++
```
in there.

So just put the line:
```
- Add metadata support
```
above that.